docs/admin/kube-scheduler.md

```diff
@@ -62,6 +62,10 @@ kube-scheduler
       --kube-api-burst=100: Burst to use while talking with kubernetes apiserver
       --kube-api-qps=50: QPS to use while talking with kubernetes apiserver
       --kubeconfig="": Path to kubeconfig file with authorization and master location information.
+      --leader-elect[=false]: Start a leader election client and gain leadership before executing scheduler loop. Enable this when running replicated schedulers.
```

Comment:
Maybe say "replicated scheduler for high availability" just to make sure people don't confuse this with running multiple schedulers for performance or partitioning diffrent scheduling algorithms.